Podcast/BookmarkViewController.swift

```diff
@@ -158,3 +152,12 @@ class BookmarkViewController: ViewController, EmptyStateTableViewDelegate, UITab
         System.endpointRequestQueue.addOperation(endpointRequest)
     }
 }
+
+extension BookmarkViewController: EpisodeDownloader {
+    func didReceive(statusUpdate: DownloadStatus, for episode: Episode) {
+        // Not worth it, this view doesn't have cells that distinguish download status
```

Comment:
agreed, you can probably delete the commented out code here

Reply:
Yeah I left it in so that later it's there when we inevitably add a download icon.